lib/commonConfig.js

```diff
@@ -245,6 +245,21 @@ function getCommandRegistry() {
     return commandRegistry;
```

Comment:
A line in the CNr

```
- Add: support NGSIv2 for Timestamp Compression plugin, through configuration (#527)
```

(Don't worry about duplication lines on #527 by the moment, we can sum up at the end but by the moment let's keep track of the changes as detailed as we can)

Reply:
Fixed in e9f6475e7ed413e8ac2e192187ffbd45399b5103